src/locales/ko/menu.ts

```diff
@@ -17,6 +17,7 @@ export const menu: SimpleTranslationEntries = {
   "username": "이름",
   "password": "비밀번호",
   "login": "로그인",
+  "orUse": "Or use",
```

Comment:
```suggestion
  "orUse": "소셜 로그인",
```

Or, if this word is too long

```suggestion
  "orUse": "혹은",
```